src/kernel/kmain.zig

```diff
@@ -169,6 +170,8 @@ fn initStage2() noreturn {
 
     tty.print("Hello Pluto from kernel :)\n", .{});
 
+    pci.init();
```

Comment:
PCI is specific to x86 so we should make the device system more abstract in the future.

Reply:
I wasn't sure of this. Does ARM have PCI?

Reply:
So the Arm architecture doesn't specify how devices are detected and such, that's all up to the board itself. The Raspberry Pi, which is what we're porting to at the moment, doesn't have PCI.